XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/Controllers/BaseController.cs

```diff
@@ -213,7 +213,7 @@ public void TryRenderControllerModel(bool useAlternatePoseAction = false)
 
             // If we didn't get an override model, and we didn't load the driver model,
             // then get the global controller model for each hand.
-            if (controllerModel.IsNull())
+            //if (controllerModel.IsNull())
```

Comment:
Why are we leaving a commented out line in?

Reply:
It is clearly described in the description of this pull request. 

> Fixed a always null warning in BaseController

Reply:
I only commented it out for now in case we planned on providing an override method to call later before this switch statement.